### PR TITLE
mitigate focus.de

### PR DESCRIPTION
**Pull request by icodebyamanda**
**Asana Task/Github Issue:** https://app.asana.com/0/0/1204862417002333/f

## Description
Adding a selector to element-hiding to mitigate focus.de blank space

### CODE REVIEW

File: features/element-hiding.json [hunk 1]
```diff
@@ -171,6 +171,10 @@
                 "selector": ".top-banner",
                 "type": "hide-empty"
             },
+            {
+                "selector": "#top-banner",
+                "type": "hide"
```
Comment (GuiltyDolphin) [inline features/element-hiding.json:176]: Do we want this to be `hide-empty` so we don't start hiding populated content?

Comment (icodebyamanda) [reply features/element-hiding.json]: yes - change made

File: features/tracker-allowlist.json [hunk 2]
```diff
@@ -1117,6 +1117,14 @@
                     }
                 ]
             },
+                    },
```
Comment (GuiltyDolphin) [inline features/tracker-allowlist.json:1120]: Uh the formatting sems weird here. Did something go awry during a rebase/merge?

There should be an existing entry for `greylabeldelivery.com`, so you can add there.

FYI, the "reason" field should link to a GitHub issue (descriptive comments are deprecated).

File: features/element-hiding.json [hunk 3]
```diff
@@ -334,7 +338,27 @@
...
+                "type": "closest-empty"
+              },
+              {
+                "selector": "[id*='M_FLYCPT_hor']",
+                "type": "closest-empty"
+              },
+              {
+                "selector": ".cls_slot_xxoutstreamxx",
+                "type": "hide-empty"
+              }
```
Comment (dharb) [inline features/element-hiding.json:361]: These are specific to focus.de, yeah? Let's move them out of the global section so they aren't applied on all pages and instead add them under the `domains` section, eg:
```JSON
{
    "domain": "focus.de",
    "rules": [
        {
            "selector": "[id*='M_CONTENTAD']",
            "type": "closest-empty"
        }
    ]
}
```

File: features/tracker-allowlist.json (hunk 2 above)
Comment (dharb) [reply features/tracker-allowlist.json:1120]: What Ben said, see https://github.com/duckduckgo/privacy-configuration/pull/949 for an example of a content blocking mitigation.

**Review by dharb — CHANGES_REQUESTED** [review]
Left a couple comments

**Review by dharb — APPROVED** [review]
Looks great, thank you!

**Pull request merged**